src/SammenligningProvider.tsx

```diff
@@ -18,10 +18,25 @@ export type Sykefraværprosent = {
     erMaskert?: boolean;
 };
 
+export enum RestSammenligningStatus {
+    'Suksess' ,
+    'LasterInn',
+    'IkkeAutorisert',
```

Comment:
Hva er forskjellen på IkkeAutorisert og HarIkkeRettigheterIAltinn?

Reply:
Åja, nå ser jeg mappingen under. Burde kanskje være "IkkeAutentisert" i stedet?

Reply:
Eller, det er vel egentlig det samme som "IkkeInnlogget"

Reply:
Ja, "IkkeInnlogget" gir mer mening, "Ikke Autorisert" kommer vel direkte fra engelsk "Not Authorized" :)